index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>make it meaningful</title>
```

Comment:
Haha, a little too literal here. Give your portfolio a title with your name and dev so that when people google you, they can find your page (once we host them down the road).